src/utils/IdealGraphVisualizer/HierarchicalLayout/src/main/java/com/sun/hotspot/igv/hierarchicallayout/HierarchicalLayoutManager.java

```diff
@@ -1458,9 +1420,8 @@ protected void run() {
 
                 // Whether the node has non-self reversed edges going downwards.
                 // If so, reversed edges going upwards are drawn to the left.
-                boolean hasReversedDown =
-                    reversedDown.size() > 0 &&
-                    !(reversedDown.size() == 1 && hasSelfEdge);
```

Comment:
Please avoid style changes to code that you did not touch otherwise (sometimes the IDE does this automatically)